smart_control/environment/environment.py

```diff
@@ -1253,10 +1247,6 @@ def _action_strings(
     t0 = time.time()
     reward_value = 0.0
     observation = None
-    last_timestamp = self.current_simulation_timestamp
```

Comment:
removing now unused variable `last_timestamp` because it is no longer passed to function that doesn't use it (https://github.com/google/sbsim/pull/32/files#r2070816466), and no longer used anywhere else